test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1243,41 +1243,36 @@ def forward(self, x):
             node_list,
         )
 
-        # Assumption: topologically sorted graph, single user for each quantize_per_tensor node
-        # Assumption is good enough for the simple model used for testing at present
-        def verify_quant_dequant_iotypes(m, activation_dtype, activation_quant_dtype):
-            quant_node = None
+        def verify_quant_dequant_iotypes_new(m):
```

Comment:
hmm, why changing the name?

Reply:
Sorry for this unintended change. Let me correct.